server/models/quizModels.js

```diff
@@ -9,7 +9,7 @@ const pool = new Pool({
 
 module.exports = {
   query: (text, params, callback) => {
-    console.log('executed query', text);
+    //console.log('executed query', text);
```

Comment:
Still needed?

Reply:
John suggested keeping it until post development.